front/app/containers/ProjectsShowPage/shared/poll/PollForm.tsx

```diff
@@ -85,13 +109,37 @@ const PollForm = ({ questions, id, type, disabled, projectId }: Props) => {
   };
 
   const sendAnswer = () => {
-    if (validate() && id) {
```

Comment:
We could actually remove this validate() call here, since the submit button will only ever be enabled if validate() passes (line 169).